tests/sles4sap/hana_install.pm

```diff
@@ -193,12 +193,14 @@ sub run {
     # Configure NVDIMM devices only when running on a BACKEND with NVDIMM
     my $pmempath = get_var('HANA_PMEM_BASEPATH', "/hana/pmem/$sid");
     if (get_var('NVDIMM')) {
-        my $nvddevs = get_var('NVDIMM_NAMESPACES_TOTAL', 2);
-        foreach my $i (0 .. ($nvddevs - 1)) {
-            assert_script_run "mkdir -p $pmempath/pmem$i";
-            assert_script_run "mkfs.xfs -f /dev/pmem$i";
-            assert_script_run "echo /dev/pmem$i $pmempath/pmem$i xfs defaults,noauto,dax 0 0 >> /etc/fstab";
-            assert_script_run "mount $pmempath/pmem$i";
+        # Read all configured pmem devices on the system
+        my @pmem_devices_all = split("\n", script_output("find /dev/pmem*"));
+        foreach my $pmem_device (@pmem_devices_all) {
+            $pmem_device =~ s:/dev/(.*):$1:;
+            assert_script_run "mkdir -p $pmempath/$pmem_device";
+            assert_script_run "mkfs.xfs -f /dev/$pmem_device";
+            assert_script_run "echo /dev/$pmem_device $pmempath/$pmem_device xfs defaults,noauto,dax 0 0 >> /etc/fstab";
+            assert_script_run "mount $pmempath/$pmem_device"; my $nvddevs = get_var('NVDIMM_NAMESPACES_TOTAL', 2);
```

Comment:
I guess ` my $nvddevs = get_var('NVDIMM_NAMESPACES_TOTAL', 2);` it's a holdover from a previous version and it can be removed, right? Otherwise, what's the purpose of the line?